spec/rubocop/cop/style/method_called_on_do_end_block_spec.rb

```diff
@@ -5,62 +5,57 @@
 
   context 'with a multi-line do..end block' do
     it 'registers an offense for a chained call' do
-      inspect_source(cop, <<-END.strip_indent)
+      expect_offense(<<-RUBY.strip_indent)
         a do
           b
         end.c
-      END
-      expect(cop.offenses.size).to eq(1)
-      expect(cop.highlights).to eq(['end.c'])
+        ^^^^^ Avoid chaining a method call on a do...end block.
```

Comment:
As before - the highlight functionality is a bit confusing for a multi-line code example, as you can't really tell what exactly are you expecting to highlight. I wonder if we can do something about this... Everything else looks good to me. I'm just wondering about cases with long code blocks and long messages.